pkg/cli/separators.go

```diff
@@ -43,34 +43,34 @@ const USV_RS_FOR_HELP = "U+241E (UTF-8 \\xe2\\x90\\x9e)"
 const DEFAULT_JSON_FLATTEN_SEPARATOR = "."
 
 var SEPARATOR_NAMES_TO_VALUES = map[string]string{
-	"ascii_esc":  ASCII_ESC,
-	"ascii_etx":  ASCII_ETX,
-	"ascii_fs":   ASCII_FS,
-	"ascii_gs":   ASCII_GS,
-	"ascii_null": ASCII_NULL,
-	"ascii_rs":   ASCII_RS,
-	"ascii_soh":  ASCII_SOH,
-	"ascii_stx":  ASCII_STX,
-	"ascii_us":   ASCII_US,
-	"asv_fs":     ASV_FS,
-	"asv_rs":     ASV_RS,
-	"colon":      COLON,
-	"comma":      COMMA,
-	"cr":         CR,
-	"crcr":       CRCR,
-	"crlf":       CRLF,
-	"crlfcrlf":   CRLFCRLF,
-	"equals":     EQUALS,
-	"lf":         LF,
-	"lflf":       LFLF,
-	"newline":    NEWLINE,
-	"pipe":       PIPE,
-	"semicolon":  SEMICOLON,
-	"slash":      SLASH,
-	"space":      SPACE,
-	"tab":        TAB,
-	"usv_fs":     USV_FS,
-	"usv_rs":     USV_RS,
+	"ascii_esc":       ASCII_ESC,
+	"ascii_etx_char":  ASCII_ETX,
+	"ascii_fs":        ASCII_FS,
+	"ascii_gs":        ASCII_GS,
+	"ascii_null_char": ASCII_NULL,
+	"ascii_rs":        ASCII_RS,
+	"ascii_soh_char":  ASCII_SOH,
+	"ascii_stx_char":  ASCII_STX,
```

Comment:
Nice! But isn't the deprecated old values mapping missing? 
Something like this?

```
    "ascii_esc":       ASCII_ESC,
    "ascii_etx":       "\\x04",     // deprecated
    "ascii_etx_char":  ASCII_ETX,
    "ascii_fs":        ASCII_FS,
    "ascii_gs":        ASCII_GS,
    "ascii_null":      "\\x01",     // deprecated
    "ascii_null_char": ASCII_NULL,
    "ascii_rs":        ASCII_RS,
    "ascii_soh":       "\\x02",     // deprecated
    "ascii_soh_char":  ASCII_SOH,
    "ascii_stx":       "\\x03",     // deprecated
    "ascii_stx_char":  ASCII_STX,
```

Sorry I should make a pr but I'm using another machine and it's a bit cumbersome from this one

Reply:
@GlassGruber TBH I'd rather surface a hard error if someone uses an old mapping that's wrong -- then they can fix their pipelines ...

Reply:
Maybe it can print a warning for the next release, then, old syntax goes away in the release after? I.e. a deprecation cycle ...

I definitely do not want to keep wrong values around forever ..... :|

Reply:
Totally makes sense!